packages/core/types/@chainlink/index.d.ts

```diff
@@ -41,6 +41,7 @@ declare module '@chainlink/types' {
    */
   export interface AdapterMetricsMeta {
     feedId: string
+    requestOrigin: string
```

Comment:
I think we should be careful and analyze the impact that this increase in cardinality will cause. The feedId label is already super high cardinality, and we've even thought about removing it altogether from the prometheus metrics. Although EAs shouldn't be hit with millions of different IPs, I think it's realistic that we get 10s or 100s of them here and we should measure how this affects our prom instances